docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+{% extends 'test.njk' %} {% block content %}
+<section>
+    <!-- Caption -->
+    <div class="demo" data-dom-diff="typography-caption">
+        <p class="hxCaption">Caption</p>
+    </div>
+</section>
+<section>
+    <!-- Error Message -->
+    <div class="demo" data-dom-diff="typography-error-message">
+        <hx-error>Error message goes here</hx-error>
+    </div>
+</section>
+<section>
+    <!-- Inline Code -->
+    <div class="demo" data-dom-diff="typography-inline-code">
+        <p>
+            Add an event listener with the
+            <code>EventTarget.addEventListener()</code> function.
+        </p>
+    </div>
+</section>
+<section>
+    <!-- Keyboard -->
+    <div class="demo" data-dom-diff="typography-keyboard">
+        <p>
+            Press
+            <kbd>Ctrl+Z</kbd> (
+            <kbd>Command+Z</kbd> on Mac) to undo the most recent action.
+        </p>
+    </div>
+</section>
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed